shop/src/main/resources/db/migration/V1_0__create_tables.sql

```diff
@@ -0,0 +1,85 @@
+CREATE TABLE IF NOT EXISTS Supplier(
+    ID INT PRIMARY KEY AUTO_INCREMENT,
+    NAME VARCHAR(255));
+
+CREATE TABLE IF NOT EXISTS ProductCategory(
```

Comment:
A better approach is to use the snake case naming convention for all your table names and column names. By default, SQL is case-insensitive and it might cause you some problems with Hibernate.